### PR TITLE
M0078DDG-437 Platform Creation build with the CLI

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: PlatformDeveloperGuide/platformCreation.rst [hunk 1]
```diff
@@ -153,11 +153,19 @@ And to declare the Legacy MicroEJ Generic Pack Net version ``9.2.3``:
 Platform Build
 ==============
 
-To build the MicroEJ Platform, perform as a regular :ref:`mmm_module_build`: 
+The MicroEJ Platform can be built either from the SDK or from the :ref:`MMM CLI <mmm_cli>`.
+To build the MicroEJ Platform from the SDK, perform as a regular :ref:`mmm_module_build`:
```
Comment (gharelMicroEJ) [inline PlatformDeveloperGuide/platformCreation.rst:157]: Shouldn't it be:
To build the MicroEJ Platform from the SDK, perform as regular :ref:`mmm_module_build`:

Comment (tdelhomenie-microej) [reply PlatformDeveloperGuide/platformCreation.rst:157]: Not sure. I would say "perform a regular Module Build" ("faites un Module Build habituel"). But maybe "regular" is useless and we should simplify to "perform a Module Build". WDYT ?

Comment (gharelMicroEJ) [reply PlatformDeveloperGuide/platformCreation.rst:157]: Sorry I meant it is either:
To build the MicroEJ Platform from the SDK, performed as a regular build
or
To build the MicroEJ Platform from the SDK, perform a regular build

Comment (tdelhomenie-microej) [reply PlatformDeveloperGuide/platformCreation.rst:157]: ok let's go with  "perform a regular Module Build" then.

Comment (gharelMicroEJ) [reply PlatformDeveloperGuide/platformCreation.rst]: Yes I agree.

**Pull request merged**